### ISSUE TITLE
package not found: php5.6-cli

### ISSUE THREAD

**Issue by kevinresol** [back2dos/travix#75]
https://travis-ci.org/back2dos/travix/jobs/294614963#L1094

any idea?

**Comment by andyli** [comment]
One has to `apt-get update` after `add-apt-repository` ;)

**Comment by sebthom** [comment]
Yes, apparently the ubuntu image used by travix does not support the `-u` option for `add-apt-repository` , which would directly update the package cache after adding a new repo.
